scio-jmh/src/test/scala/com/spotify/scio/jmh/BigQueryBenchmark.scala

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Actually we shouldn't check this file in since it doesn't benchmark both versions and show the difference. As long as we can use it to verify that the change doesn't degrade performance.